github/ContentFile.py

```diff
@@ -93,6 +93,18 @@ def html_url(self):
         self._completeIfNotSet(self._html_url)
         return self._html_url.value
 
+    @property
+    def last_modified(self):
+        """
+        :type: str
+        """
+        self._completeIfNotSet(self._last_modified)
```

Comment:
This is incorrect -- since last_modified isn't a property that we set in useAttributes(), don't call it like that. If anything, call it on self._path.

Reply:
Ok. Will update.